places/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.urls import path
+from places.views import PlaceReviewListView
+
+urlpatterns = [
+    path('/placereviewlist/<int:place_id>', PlaceReviewListView.as_view())
```

Comment:
지금 작성하신 엔드포인트 주소는 ```:8000/places/placereviewlist/1```입니다.
작성하신 주소만 보면 reviewlist에서 1번을 가져오는 것 처럼 보여집니다.
따라서 ```:8000/places/1/review```가 더 좋아보입니다.

```suggestion
    path('/<int:place_id>/review', PlaceReviewListView.as_view())
```